prboom2/src/r_drawflush.inl

```diff
@@ -45,6 +43,86 @@
 //
 static void R_FLUSHWHOLE_FUNCNAME(void)
 {
+   // Scaled software fuzz algorithm
+#if (R_DRAWCOLUMN_PIPELINE & RDC_FUZZ)
+    dboolean cutoff = false;
+
+    if ((temp_x + startx) % fuzzcellsize)
+    {
+        return;
+    }
+
+    int yl = tempyl[temp_x - 1];
+    int yh = tempyh[temp_x - 1];
+
+    if (!yl)
+    {
+        yl = 1;
+    }
+
+    if (yh == viewheight - 1)
+    {
+        yh = viewheight - 2;
+        cutoff = true;
+    }
+
+    int count = yh - yl + 1;
+
+    if (count < 0)
+    {
+        return;
+    }
+
+#ifdef RANGECHECK
+    if ((unsigned)x >= video.width || yl < 0 || yh  >= video.height)
+    {
+        I_Error("R_DrawFuzzColumn: %i to %i at %i", yl, yh , x);
+    }
+#endif
+
+    ++count;
+
+    byte *dest = drawvars.topleft + yl * drawvars.pitch + startx + temp_x - fuzzcellsize;
+
+    int lines = fuzzcellsize - (yl % fuzzcellsize);
+
+    do
+    {
+        count -= lines;
+
+        // if (count < 0)
+        // {
+        //    lines += count;
+        //    count = 0;
+        // }
+        const int mask = count >> (8 * sizeof(mask) - 1);
+        lines += count & mask;
+        count &= ~mask;
+
+        const byte fuzz =
+            fullcolormap[6 * 256 + dest[drawvars.pitch * fuzzoffset[fuzzpos]]];
+
+        do
+        {
+            memset(dest, fuzz, fuzzcellsize);
+            dest += drawvars.pitch;
+        } while (--lines);
+
+        ++fuzzpos;
+
+        // Clamp table lookup index.
+        fuzzpos &= (fuzzpos - FUZZTABLE) >> (8 * sizeof(fuzzpos) - 1); // killough 1/99
+
+        lines = fuzzcellsize;
+    } while (count);
+
+    if (cutoff)
```

Comment:
We probably don't need the `cutoff` code. It is left over from the old Woof implementation.

Reply:
Right, I assumed I was missing something about it. removed